lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -338,6 +347,27 @@ def write(chunk)
       # Group the entries since we have to make one call per tag.
       grouped_entries = {}
       chunk.msgpack_each do |tag, *arr|
+        tag = tag.to_s
```

Comment:
Do you mean when `require_valid_tags` is false? I guess you meant when `require_valid_tags` is true, we should not do `tag.to_s` because any non-string tags should be rejected right away? I've modified it to be this way.